### DISCUSSION TITLE
Import games in a folder and turn the folder into a Game View

### DISCUSSION THREAD

**Discussion by CommodoreOS** [lantzelot-swe/PCUGameManager#10] [Ideas]
Lantzelot... Is it possible in PCUAE Manager when you import games that are in the carousel format, it imports the folder the games are in too so it makes a Game View with the same name as the folder is named so like if I have games in a folder called `C64_0_to_G` it imports the name of the folder and makes a Game View from it called `C64_0_to_G` and then imports the games from that folder into that Game View.
I would like to be able to just point the importer to a folder and it with import the games and if any of the games are in folders add them as Game Views and add the games in each folder to that view...?
so If I wanted to add the Games folder, I just point it to the Game folder and PCUAE Manager will add the games and make the folders as Game views and then add the games to each view how the games are set in each folder.

**Comment by lantzelot-swe** [comment]
Yes, I can add an option to create a game view during import. The gameview have to use the "viewtag" property for filtering and all imported games can be tagged with the same tag. I can look into it.
